vpn/src/main/java/com/duckduckgo/mobile/android/vpn/apps/TrackingProtectionAppsRepository.kt

```diff
@@ -119,11 +124,21 @@ class RealTrackingProtectionAppsRepository @Inject constructor(
         ddgExclusionList: List<AppTrackerExcludedPackage>,
         userExclusionList: List<AppTrackerManualExcludedApp>
     ): Boolean {
+        if (transparencyModeBugFixForAndroid12(appInfo)) {
+            Timber.d("DDG App in transparency mode for Android 12")
+            return false
+        }
         return VpnExclusionList.isDdgApp(appInfo.packageName) ||
             isSystemAppAndNotOverriden(appInfo) ||
             isManuallyExcluded(appInfo, ddgExclusionList, userExclusionList)
     }
 
+    // https://issuetracker.google.com/issues/217570500
+    // https://app.asana.com/0/1174433894299346/1201657419006650
+    private fun transparencyModeBugFixForAndroid12(appInfo: ApplicationInfo): Boolean {
+        return appBuildConfig.sdkInt > Build.VERSION_CODES.S && VpnExclusionList.isDdgApp(appInfo.packageName) && appBuildConfig.flavor == INTERNAL
```

Comment:
Ditto, `>=S` otherwise it doesn't work

```suggestion
        return appBuildConfig.sdkInt >= Build.VERSION_CODES.S && VpnExclusionList.isDdgApp(appInfo.packageName) && appBuildConfig.flavor == INTERNAL
```